bin/reload

```diff
@@ -19,6 +19,8 @@ show_run("Create schema", "app/console doctrine:schema:create " . $stringOptions
 
 show_run("Cache clear", "php app/console cache:clear " . $stringOptions);
 
+show_run("Create media dir", "mkdir web/uploads/media");
```

Comment:
merge develop. media folder not needed now. Even you need some folder you can user .gitkeep file for this